src/pages/home/report/ReportActionsView.js

```diff
@@ -153,9 +153,13 @@ class ReportActionsView extends React.Component {
         Report.fetchActions(this.props.reportID);
 
         const copyShortcutConfig = CONST.KEYBOARD_SHORTCUTS.COPY;
-        this.unsubscribeCopyShortcut = KeyboardShortcut.subscribe(copyShortcutConfig.shortcutKey, () => {
-            this.copySelectionToClipboard();
-        }, copyShortcutConfig.descriptionKey, copyShortcutConfig.modifiers, false);
+        this.unsubscribeCopyShortcut = KeyboardShortcut.subscribe(
+            copyShortcutConfig.shortcutKey,
+            this.copySelectionToClipboard,
```

Comment:
**question:** Do we need to bind this in the constructor since we are refactoring the arrow function?

Reply:
Doesn't look like the function is referencing `this`, so it looks like we don't need to in this case.

Reply:
**separate thought:** Why do we need a keyboard event for copying and pasting text? 😵‍💫 

Maybe we can add some context to the `copySelectionToClipboard()` JSDoc.

Reply:
Yeah, Rory is correct.

As for why it's needed, it came from this PR https://github.com/Expensify/App/pull/6732 and it's so that markdown formatting is preserved when copying selected text.

Which indicates that I need to update the testing instructions 😁